tsconfig.json

```diff
@@ -2,6 +2,8 @@
   "extends": "@toruslabs/config/tsconfig.default.json",
   "compilerOptions": {
     "baseUrl": "./",
+    "allowJs": true,
+    "skipLibCheck": true,
```

Comment:
Why does it need skipLibCheck?

Reply:
You are right, I don't think it is needed. Removed `skipLibCheck`.

For context:
I copied these two lines from `tss-server/tsconfig.json` because I noticed that there the types were packaged correctly and these two lines were different over there. Maybe `skipLibCheck` should also be removed there?

Reply:
Interesting. Perhaps it could be removed there as well, we'd have to check.